<?xml version = "1.0" encoding="UTF-8"?>
<launch>
	<node pkg="pioneer_run" name="pioneer_controller" type="pioneer_run_node" output="screen">
	
		<param name="world_frame_id" value="/world"/>
		<param name="base_frame_id" value="/vicon/pioneer/pioneer"/>
		<param name="linear_gain/P" value="0.3"/>
		<param name="linear_gain/I" value="0.1"/>
		<param name="angular_gain" value="0.2"/>
		<param name="angular_tangential_gain" value="0.2"/>
		<param name="control_input_topic" value="/cmd_vel"/>
		<param name="goal_topic" value ="target_goal"/>
		
	</node>
	<node pkg="pioneer_run" name="pioneer_ref_talker" type="path_talker.py" output="screen">
		<param name="ref_traj_topic_name" value="/pioneer_controller/target_goal" type="str"/>
		<param name="bagfile_name" value="$(find pioneer_run)/scripts/2018-09-06-20-48-16.bag" type="str"/>
		<param name="target_name" value="target"/> <!--In gazebo env-->
	</node>

</launch>

